packages/dotnet-sdk/src/TeamsFx.Test/Conversation/CommandResponseMiddlewareTest.cs

```diff
@@ -0,0 +1,150 @@
+﻿namespace Microsoft.TeamsFx.Test.Conversation
+{
+    using Microsoft.Bot.Builder;
+    using Microsoft.Bot.Schema;
+    using Microsoft.TeamsFx.Conversation;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Moq;
+    using System;
+    using System.Threading.Tasks;
+
+    [TestClass]
+    public class CommandResponseMiddlewareTest
+    {
+        private readonly CommandResponseMiddleware _middleware;
+        private readonly StringTrigger _stringTrigger = new("SampleTest");
+        private readonly RegExpTrigger _regexTrigger = new(@"^test (.*?)$");
+
+        public CommandResponseMiddlewareTest()
+        {
+            _middleware = new CommandResponseMiddleware();
+        }
+
+        [TestMethod]
```

Comment:
Consider [DataTestMethod] to cover more cases (case-insensitive, extra-spaces, special chars, etc.)?

Reply:
Sounds good, will update later.